packages/did/src/cbor-web.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'cbor-web'
```

Comment:
I'm debating - is an empty declaration file better than a `@ts-expect-error` in front of the import? The result is the same - it's treated as any

Reply:
I don't think that's the solution, this way we'll still keep the dependency on cbor - extracting type definitions from another package and adding it to your build is not really what tsc is made for I think

Reply:
[Here’s a thing by the way](https://github.com/hildjj/node-cbor/commit/bb3787a11f7ad6d8508574b91e293be825765430)

Reply:
[And its discussion](https://github.com/hildjj/node-cbor/pull/177)